data/completion/complete.sh

```diff
@@ -0,0 +1,46 @@
+# -*- bash -*-
+
+_complete_from_snap() {
+    {
+        read -a opts
+        read bounced
+        read sep
+        if [ "$sep" ]; then
```

Comment:
my personal problem with `-n` is that i always have to look it up as i never remember which is it and which is `-z`.